essentials.yml

```diff
@@ -0,0 +1,12 @@
+---
+ath:
+  athRevision: "dad333092159cb368efc2f9869572f0a05d255ac"
+  tests:
+    - "GitPluginTest"
+    - "GitUserContentTest"
+    - "MultipleScmsPluginTest"
+    - "WorkflowPluginTest#hello_world_from_git"
+    - "WorkflowPluginTest#testSharedLibraryFromGithub"
```

Comment:
The current UC provider for ATH relies on the general public `update-center.json`. Since that changes from hour to hour, a release of (say) `workflow-multibranch` could cause _this_ plugin’s PRs to abruptly begin failing, which is no good as I have argued in https://github.com/jenkinsci/jep/pull/92 and elsewhere. Would be solvable by the WAR Packager, I think, if given deterministic input.

Reply:
And what is the value to test against an old version of {{workflow-multibranch}} in a PR?, in the case that you comment, and I agree is possible, IMO the PR has fulfilled its objective of finding a problem with the latest released version of {{workflow-multibranch}} and should be fixed.

Once all the work related to JEP-305 is in place and there is a way to deterministically  use the latest versions of plugins under test I heartily agree that CWP is the way to go, that is the reason I have created JENKINS-51008

Reply:
I think it can be merged now. We are not making any irreversible decisions here.

Reply:
I'd like to evaluate it inside my testing infrastructure before it is merged.

I looked at the ci.jenkins.io build results and didn't see any indication that this new code was executed.  Is it being executed and I missed it?  

If not, is there a way to have it run before the merge to master?

Reply:
FTR

    Loading trusted files from base branch master at 44e9eb0716165de33f277fd2552d6a4de2961bb0 rather than b53644480cc0e098719a987805eb8f5a14a66ef7